abcclassroom/example-data/sample_config.yml

```diff
@@ -0,0 +1,56 @@
+
+# The GitHub organization to host your GitHub classroom
+# Create a new organization on GitHub and change this to the name you gave it
+organization: earth-analytics-edu
```

Comment:
@nkorinek  let's do the following.

1. please name this config.yml rather than sample_config.yml
2. rather than having org names here use your-github-org-name-here